core-bundle/contao/templates/twig/backend/data_container/operations.html.twig

```diff
@@ -1,20 +1,43 @@
-{% for operation in operations %}
-    {{ _self.operation(operation) }}
-{% endfor %}
+{% set has_submenu = false %}
+<nav>
```

Comment:
_Might be an accessibility concern_
I am unsure of the usage of `nav` here since it's usually used for major blocks of navigation links.
This would create lots of navigations on every page when there's many context-menus being rendered.